docs/learn/dapp-developer/register-lsp12-issued-assets.md

```diff
@@ -0,0 +1,204 @@
+  <TabItem value="web3js" label="web3.js">
+
+```javascript
+import Web3 from 'web3';
+import { ERC725 } from '@erc725/erc725.js';
+import LSP12Schema from '@erc725/erc725.js/schemas/LSP12IssuedAssets.json';
+
+import UniversalProfileArtifact from '@lukso/lsp-smart-contracts/artifacts/UniversalProfile.json';
+import { INTERFACE_IDS } from '@lukso/lsp-smart-contracts';
+
+// We will register the issued assets by setting the following LSP12 data keys
+// - LSP12IssuedAssets[]
+// - LSP12IssuedAssetsMap:<asset-address>
+
+// add the type of asset (LSP7 or LSP8) and their address in the object list below
+const issuedAssets = [
+  {
+    interfaceId: INTERFACE_IDS.LSP7DigitalAsset,
+    address: '0xf7056bdE90f494F55967858F1e9E4AFB1026C5C8',
+  },
+  {
+    interfaceId: INTERFACE_IDS.LSP8IdentifiableDigitalAsset,
+    address: '0xf651b88925C0B6C81Ad6f658a2F104226d837F60',
+  },
+  //   {
+  //     interfaceId: LSP7 or LSP8 interface ID (or other),
+  //     address: '0xasset-address',
+  //   },
+];
+
+const web3 = new Web3(window.lukso);
+await web3.eth.requestAccounts();
+const accounts = await web3.eth.getAccounts();
+```
+
+  </TabItem>
+  <TabItem value="ethersjs" label="ethers.js">
+
+```javascript
+import { ethers } from 'ethers';
+
+import { ERC725 } from '@erc725/erc725.js';
+import LSP12Schema from '@erc725/erc725.js/schemas/LSP12IssuedAssets.json';
+
+import UniversalProfileArtifact from '@lukso/lsp-smart-contracts/artifacts/UniversalProfile.json';
+import { INTERFACE_IDS } from '@lukso/lsp-smart-contracts';
+
+// We will register the issued assets by setting the following LSP12 data keys
+// - LSP12IssuedAssets[]
+// - LSP12IssuedAssetsMap:<asset-address>
+
+// add the type of asset (LSP7 or LSP8) and their address in the object list below
+const issuedAssets = [
+  {
+    interfaceId: INTERFACE_IDS.LSP7DigitalAsset,
+    address: '0xf7056bdE90f494F55967858F1e9E4AFB1026C5C8',
+  },
+  {
+    interfaceId: INTERFACE_IDS.LSP8IdentifiableDigitalAsset,
+    address: '0xf651b88925C0B6C81Ad6f658a2F104226d837F60',
+  },
+  //   {
+  //     interfaceId: LSP7 or LSP8 interface ID (or other),
+  //     address: '0xasset-address',
+  //   },
+];
+
+const provider = new ethers.BrowserProvider(window.lukso);
+await provider.send('eth_requestAccounts', []);
+const myWallet = await provider.getSigner();
+```
+
+  </TabItem>
+</Tabs>
+
+## Encode the data keys
+
+Use `erc725.js` nd LSP12 JSON schema to encode the data keys related to `LSP12IssuedAssets[]`.
+
+```javascript
+const erc725 = new ERC725(
+  LSP12Schema,
+  UNIVERSAL_PROFILE_ADDRESS,
+  RPC_ENDPOINT,
+  {
+    ipfsGateway: 'https://api.universalprofile.cloud/ipfs',
+  },
+);
+
+const allAssetAddresses = issuedAssets.map((asset) => asset.address);
+
+const issuedAssetsMap = issuedAssets.map((asset, index) => {
+  return {
+    keyName: 'LSP12IssuedAssetsMap:<address>',
+    dynamicKeyParts: asset.address,
+    value: [asset.interfaceId, ERC725.encodeValueType('uint128', index)],
```

Comment:
Might be good for the user to add a little comment to explain what `encodeValueType('uint128', index)` does